File: pigatize.rb
def pigatize(word)
  if starts_with_vowel?(word)
    "#{word}way"
  else
    "#{word[1, word.length]}#{word[0]}ay"
  end
end

def starts_with_vowel?(word)
  word[0] =~ /a|e|i|o|u|A|E|I|O|U/
end


puts pigatize("kate")
puts pigatize("end")
puts pigatize("Darron")
puts pigatize("Aaron")

File: pigatize.js
function Pigatize(word){
  var word = "Kate";
  var index;
  index = word.search(/aeiouAEIOU]/);
  if (index <= 0) {
    console.log(word + "way");
  }
  else {
    console.log(word.substring(index, word.length) + word.substring(0,index) = "ay");
  }
  }

